src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -44,6 +44,7 @@ public void Dispose()
 
         [Theory]
         [ClassData(typeof(SslProtocolSupport.UnsupportedSslProtocolsTestData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19458")]
```

Comment:
This should be reverted.  It was addressed by https://github.com/dotnet/corefx/pull/19477.